es6/index.js

```diff
@@ -70,18 +70,18 @@ PrerenderSPAPlugin.prototype.apply = function (compiler) {
       })
       // Backwards-compatibility with v2 (postprocessHTML should be migrated to postProcess)
       .then(renderedRoutes => this._options.postProcessHtml
-        ? renderedRoutes.map(renderedRoute => {
+        ? Promise.all(renderedRoutes.map(renderedRoute => {
```

Comment:
Yeah, it's reasonable that new features are not added to deprecated ones! :+1: 

Regarding the return of `postProcessHtml` it's value is only being used directly when it is a string, but since it is a Promise it's that promise what is returned, going down the Promise chain, which will eventually fail if someone resolves to plain html so yeah, not a good idea either.